libs/openFrameworks/graphics/ofPixels.cpp

```diff
@@ -332,6 +332,40 @@ void ofPixels_<PixelType>::setFromAlignedPixels(const PixelType * newPixels, int
 }
 
 template<typename PixelType>
+void ofPixels_<PixelType>::setFromAlignedPixels(const PixelType * newPixels, int width, int height, ofPixelFormat _pixelFormat, std::vector<int> strides) {
+	int channels = channelsFromPixelFormat(_pixelFormat);
+	if(channels==0) return;
+
+	if(width*channels==strides[0]){
```

Comment:
this doesn't make much sense for I420 since it doesn't have channels. something like:

```
if(width==strides[0] && width/2==strides[1] && width/2==strides[2])
```

would be a better check